corehq/util/log.py

```diff
@@ -304,6 +305,17 @@ def draw(position):
     print("Elapsed time: {}".format(display_seconds((end - start).total_seconds())))
 
 
+def get_traceback_string():
+    if six.PY3:
+        from io import StringIO
+        f = StringIO()
```

Comment:
Is there a particular reason you can't use this in python 2?

Reply:
When I tried using ```io.StringIO``` I would get an issue with bytes, and when I tried ```io.BytesIO``` I would get an issue with unicode.  This seemed like the next best solution.

Reply:
do you foresee any issues when switching to py3 later? seems like switching the library could bring up some bugs

Reply:
I've tested the code in py3 and it works.  The ```get_traceback_string``` should have the same output in py2/py3 (a string) so I don't expect anything to break since all of the differences are encapsulated in this function.